docs/VCIssuance-vs-DataProvider.md

```diff
@@ -0,0 +1,28 @@
+# VCIssuance vs DataProvider
+
+
+Certify has a plugin model to issue VCs so that implementors can extend Inji Certify to issue various VCs and and the plugin can be of two types.
+
+1. VCIssuancePlugin
+2. DataProviderPlugin
+
+While they are two types of Issuing Plugins, both types are issuing VCs by connecting to a configured datasource. The two types of Plugins enable Inji Certify to operate differ in where VC Signing happens.
+
+## How to choose to implement either one?
+
+- An integrator can choose to implement VCIssuancePlugin interface if they want to implement the VC Signing by themselves. This gives more power to the VC Plugin authors in choosing to support their own formats, signing algorithms which may or may not be supported by Certify.
```

Comment:
We don't want to glorify this option. we should put this simply saying that if they already have a solution which generates VCs and they want certify to only do the OpenID4VCI based delivery of VC, then they can use this option. Just keep it at that.